_pages/clients/minds.com.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Minds.com
+permalink: /minds.com/
+web:
+github: Minds/nostr-nips
+instances:
+- name: Minds.com
+  url: https://minds.com
+  comment: Minds is a social network with plans integrate a nostr relay and functions in their social network.  
+tags:
```

Comment:
The minds client is not public source. The GH repo is of the nostr standards called NIPs. For projects that have only a single instance permanently, that instance is usually the `web: `.

```suggestion
web: https://minds.com
github: 
instances:
tags:
```